packages/blog/tailwind.config.js

```diff
@@ -0,0 +1,40 @@
+module.exports = {
+  purge: [],
+  darkMode: false, // or 'media' or 'class'
+  theme: {
+    extend: {
+      fontFamily: {
+        sans: [
+          "Montserrat",
```

Comment:
overriden the default values and set `Montserrat` and `Merriweather` as default for sans and serif respectively